packages/lit-dev-content/samples/tsconfig.json

```diff
@@ -10,16 +10,9 @@
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
     "types": [],
+    "rootDir": ".",
+    "outDir": "js"
   },
-  "include": [
-    "./**/*.ts",
-  ],
-  "exclude": [
-    "node_modules/**/*",
-    // TODO(kschaaf): Remove once lit released with these directives
-    "examples/directive-async-append/**/*",
-    "examples/directive-async-replace/**/*",
-    // (arthure) This tutorial step is intentionally incomplete
-    "tutorial/06-template-logic/before/**/*",
```

Comment:
Clarifying my understanding. This was being skipped since originally there was a type error (i.e. missing `addToDo` method in this example.
You've fixed the error by adding a stub so can remove from the exclude. And we want this to be "type valid" code such that we can transform it into javascript?

Reply:
Exactly

Reply:
Clarifying: we could actually still emit the code even with a type checking error, but it could be a bit complicated to distinguish between intentional vs unintentional errors. We want the build to fail if you introduce an unintentional type error. Given that this was the *only* case like that in the entire site, it seemed reasonable to just make this one spot type-correct, since it doesn't affect that tutorial step too much.

Reply:
Thank you for the answer! I really love that the build fails in the error case.

Reply:
> Out of scope questions: Is there a way to manually test the JS variant of the tutorial? Does the solve button work the same?

In the next PR, I'll integrate a JS/TS switch into all the playgrounds, including the tutorial. I'll put the switch behind a `?mods` parameter. Then we'll be able to test the tutorial along with all the inline docs samples.

> Do the code samples in the inline tutorial exposition get transformed to JS (Asking since I'm not sure if the code snippets in the tutorial exposition are references to live samples or duplicated in the content)?

Inline code snippets won't get automatically transformed, since they are usually not complete programs (e.g. just the content of a class, which won't parse). Instead I'm going to add a mechanism for defining both a TS and JS version of any snippet inside the markdown, and we'll just show/hide one or the other depending on the setting.